src/sass/foundation/variables.scss

```diff
@@ -42,7 +42,7 @@ $grayLightLight:    #9e9ea2 !default;
 $grayLight:         #c5c3c7 !default;
 $grayLighter:       #d9d8dd !default;
 $white:             #FFFFFF !default;
-
+$transparent:       rgba(0,0,0,0) !default;
```

Comment:
I should be this but less2sass is to dumb to understand that is a named color and try to apply toHSL on it and then, fail to compile everything...